include/interface/mouse.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+
+namespace Input
+{
+
+//TODO: rename those to btn_*
+enum Button { Left = 0, Right, Middle, Count};
+
+enum Cursor {
+    Cursor_None = -1,
```

Comment:
Same stuff, upper case for constants. 
```suggestion
    CURSOR_NONE = -1;
    ...
```